packages/core-js/internals/advance-string-index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
Please, add a comment that it's `AdvanceStringIndex` abstract operation and the link to the spec.